omaha/main.scons

```diff
@@ -65,12 +65,12 @@ _is_google_update_build = False
 # These values must be changed when customizing open source Omaha.
 # You may also wish to replace the language-neutral constants in the individual
 # *generated_resources_*.rc files with localized strings.
-_FULL_COMPANY_NAME = 'Dental Wings Inc.'
-_SHORT_COMPANY_NAME = 'DW'
-_PRODUCT_NAME = 'DentalWingsUpdate'
+_FULL_COMPANY_NAME = 'StraumannGroup Inc.'
```

Comment:
shouldn't it be Straumann Group? (with a space)

Reply:
Done